tokenserver/src/server.js

```diff
@@ -51,8 +51,10 @@ expressApp.use(errorHandlingMiddleware);
 
 
 githubApp.on("token", async ({ token, octokit, expiresAt }) => {
-  const { data } = await octokit.request("GET /user");
```

Comment:
Don't you need to keep this?

Reply:
No this just gets the username which was printed as debug information it's not required as GitHub uses the token to authenticate the user.

Reply:
Ah, I see.